docs/developers/packaging.md

```diff
@@ -118,10 +118,10 @@ specs: # specs for the 'base'  environment
   - mamba    # we add these to manage different napari versions
   - pip      # we add these to manage different napari versions
 extra_envs:
-  napari-0.4.16: # this is the environment that will actually contain the napari packages
+  {{ 'napari-NAPARI_VER'.replace('NAPARI_VER', napari_version) }}: # this is the environment that will actually contain the napari packages
```

Comment:
Alas, these won't work because they're in a code block.
I *think* it might work if you wrap the whole code block in {{ }} and quotes and then use string replacement?
Otherwise, I'm fine with this being hard-coded, as an example file.
Just in the leadin could be:
`Roughly, we will build this configuration file for napari 0.4.17:`
and then use 0.4.17
If the config changes in the future, we can update, but as an example it should be fine.
The other places are more important because a user could want to see them?